templates/docker-compose.yml

```diff
@@ -20,20 +20,44 @@ services:
       - "127.0.0.1:{{ lemmy_ui_port }}:1234"
     restart: always
     environment:
-      - LEMMY_INTERNAL_HOST=lemmy:8536
-      - LEMMY_EXTERNAL_HOST={{ domain }}
-      - LEMMY_HTTPS=true
+      - LEMMY_UI_LEMMY_INTERNAL_HOST=lemmy:8536
+      - LEMMY_UI_LEMMY_EXTERNAL_HOST={{ domain }}
+      - LEMMY_UI_HTTPS=true
     volumes:
       - ./volumes/lemmy-ui/extra_themes:/app/extra_themes
     depends_on: 
       - lemmy
 
   postgres:
-    image: postgres:12-alpine
+    image: postgres:15-alpine
     environment:
       - POSTGRES_USER=lemmy
       - POSTGRES_PASSWORD={{ postgres_password }}
       - POSTGRES_DB=lemmy
+    command: [
+      "postgres",
+      # Tune your settings via
+      # https://pgtune.leopard.in.ua/#/
+      # You can use this technique to add them here
+      # https://stackoverflow.com/a/30850095/1655478
+      "-c", "max_connections=200",
+      "-c", "shared_buffers=3GB",
+      "-c", "effective_cache_size=9GB",
+      "-c", "maintenance_work_mem=768MB",
+      "-c", "checkpoint_completion_target=0.9",
+      "-c", "checkpoint_timeout=86400",
+      "-c", "wal_buffers=16MB",
+      "-c", "default_statistics_target=100",
+      "-c", "random_page_cost=4",
+      "-c", "effective_io_concurrency=2",
+      "-c", "work_mem=4GB",
+      "-c", "min_wal_size=1GB",
+      "-c", "max_wal_size=30GB",
+      "-c", "max_worker_processes=4",
+      "-c", "max_parallel_workers_per_gather=2",
+      "-c", "max_parallel_workers=4",
+      "-c", "max_parallel_maintenance_workers=2"
```

Comment:
Are these settings a good default for all servers? Otherwise it would be better to define it in a different file which can be edited by the user without creating git conflicts.

Reply:
I'll remove... doing this as an external postgres settings file is a better way to do it anyway.